### PR TITLE
upgrade to ibm-fhir-server 4.10.2 and smart-keycloak 0.3.0

### PR DESCRIPTION
**Pull request by lmsurpre**
Signed-off-by: Lee Surprenant <lmsurpre@us.ibm.com>

### CODE REVIEW

File: data-access/fhirschema.log [hunk 1]
```diff
@@ -0,0 +1,8 @@
+2021-12-03 14:37:18.042 00000001 WARNING   com.ibm.fhir.schema.app.Main Only the APP schema is supported for Apache Derby; ignoring the passed schema name 'FHIRDATA' and using APP.
```
Comment (ibmatc) [inline data-access/fhirschema.log:1]: What is the purpose of delivering this log file?

Comment (lmsurpre) [reply data-access/fhirschema.log:1]: shouldn't have been included...i actually thought i had added it to my gitignore...fixing now

Comment (lmsurpre) [reply data-access/fhirschema.log]: done

**Review by ibmatc — APPROVED** [review]
LGTM

**Review by ranum — APPROVED** [review]
LGTM

**Pull request merged**